packages/host/app/components/operator-mode/definition-container.gts

```diff
@@ -22,6 +22,8 @@ interface Signature {
     realmIconURL: string | null | undefined;
     variant: DefinitionVariant;
     isActive: boolean;
+    url?: URL;
+    onSelectDefinition?: (newUrl: URL) => void;
     infoText?: string;
     onCreate?: () => void;
```

Comment:
just in general--for this speculative code that doesn't do anything yet--lets not add it. its really hard to figure out your intent when the code doesn't do anything. for instance, i see htat we don't show buttons when handlers are not sent in. but i don't see the logic for when we do and dont' send in handlers--all i see is a sample duplicate handler being sent if (which by the way is being removed). In general--just don't write speculative code unless you have a good reason. I live by the mantra: "The winds of tomorrow will blow tomorrow" :wink:

Reply:
it's better to just add `notImplmented()` functions right here in this module for all these buttons instead of threading thru actions in the arguments that don't do anything yet. Trust that the next developer that picks up this work will do the right thing.